projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -724,6 +724,19 @@ export interface FeatureTogglesInterface {
    * Enabling this flag removes the margin animation to improve performance and user experience.
    */
   disableCxPageSlotMarginAnimation?: boolean;
+
+  /**
+   * Feature flag to enable using <link rel=preconnect> in the index.html.
+   *
+   * ## Why this flag exists
+   * 1. **Preconnecting is not needed (and won't be performed) if the domain of the media base url is the same as the storefront's domain.
+   *
+   * ## When enabled:
+   * Adding rel=preconnect to a <link> informs the browser that your page intends to establish a connection to another domain,
+   * and that you'd like the process to start as soon as possible. Resources will load more quickly because the setup process
+   * has already been completed by the time the browser requests them.
+   */
```

Comment:
This section now looks a bit strange with the title `Why this flag exists`, doesn't it?:
```
   * ## Why this flag exists
   * 1. **Preconnecting is not needed (and won't be performed) if the domain of the media base url is the same as the storefront's domain.
```
I suggest to move it to the bottom Note.

```suggestion
  /**
   * Feature flag to enable using <link rel=preconnect> in the index.html.
   *
   * ## When enabled:
   * Adding rel=preconnect to a <link> informs the browser that your page intends to establish a connection to another domain,
   * and that you'd like the process to start as soon as possible. Resources will load more quickly because the setup process
   * has already been completed by the time the browser requests them.
   *
   * Note: Preconnecting is not needed (and won't be performed) if the domain of the media base url is the same as the storefront's domain.
   */
```